runtime/onert/core/src/util/shapeinf/StridedSlice.cc

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
```

Comment:
If some portion of the code came from TensorFlow, please add copyright from it.

Reply:
I added copyright.